tests/test_utils.py

```diff
@@ -405,6 +405,55 @@ def test_concat(objs, overwrite, expected):
         pd.testing.assert_frame_equal(obj, expected)
 
 
+@pytest.mark.parametrize(
+    'obj, expected_duration',
+    [
+        (
+            audformat.segmented_index(),
+            pd.Timedelta(0, unit='s'),
+        ),
+        (
+            audformat.segmented_index(['f1'], [0], [2]),
+            pd.Timedelta(2, unit='s'),
+        ),
+        (
+            audformat.segmented_index(['f1'], [0.1], [2]),
+            pd.Timedelta(1.9, unit='s'),
+        ),
+        (
+            audformat.segmented_index(['f1', 'f2'], [0, 1], [2, 2]),
+            pd.Timedelta(3, unit='s'),
+        ),
+        (
+            audformat.segmented_index(['f1'], [0]),
+            pd.Timedelta(np.NaN, unit='s'),
```

Comment:
Calling duration on an segmented index with `start=0`, `end=NaT`, should return `NaN` or `NaT` or something like this.
This test worked before, but know `duration()` returns `Timedelta('0 days 00:00:00')` instead.

Reply:
This is now fixed, but calling `audiofile.duration()` when `end=NaT` is given.